collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from collections import Counter
+from random import SystemRandom
+import requests
+from bases.FrameworkServices.SimpleService import SimpleService
```

Comment:
Use `UrlService` class as a base class. Then you don't need `requests`.

Reply:
done